venv_update.py

```diff
@@ -540,7 +540,7 @@ def stage1(venv_path, reqs):
         return 'virtualenv executable not found: %s' % python
 
     # ensure that a compatible version of pip is installed
-    run(('pip', '--version'))
+    run((python, '-m', 'pip.__main__', '--version'))
```

Comment:
I assume this is meant to be checking the version of pip inside the virtualenv, right? We don't really care what version is available outside the virtualenv afaict.